src/libs/EmojiUtils.js

```diff
@@ -116,12 +136,11 @@ function mergeEmojisWithFrequentlyUsedEmojis(emojis, frequentlyUsedEmojis = [])
 
     let allEmojis = [{
         header: true,
-        code: 'Frequently Used',
+        code: 'frequentlyUsed',
     }];
 
-    allEmojis = allEmojis.concat(getDynamicSpacing(allEmojis.length));
-    allEmojis = allEmojis.concat(frequentlyUsedEmojis, getDynamicSpacing(frequentlyUsedEmojis.length));
-    allEmojis = allEmojis.concat(emojis);
+    allEmojis = allEmojis.concat(frequentlyUsedEmojis, emojis);
+    allEmojis = addSpacesToEmojiCategories(emojis);
     return allEmojis;
 }
 
```

Comment:
Export could be cleaned up.
`trimEmojiUnicode`, `getDynamicSpacing` aren't used anywhere else.